exercises/practice/binary-search-tree/.meta/config.json

```diff
@@ -1,6 +1,13 @@
 {
   "blurb": "Insert and search for numbers in a binary tree.",
-  "authors": [],
+  "authors": [
+    "amscotti"
```

Comment:
```suggestion
    "amscotti",
    "devkabiir"
```

Reply:
I agree with this change. @ErikSchierboom the commit that this script got amscotti from listed this:

> amscotti authored and Stargator committed

But what happened was that amscottii made a commit after @devkabiir created the exercise. He just formatted a line. My name is there because I approved and merged the commit, but I didn't do any actual work.

This is why I asked about how the script differentiated between someone that made the change (commit) and the person that pushed it or merged it.